### DISCUSSION TITLE
Change default bottles location?

### DISCUSSION THREAD

**Discussion by EnfrigFalco** [bottlesdevs/Bottles#249] [Q&A] [answered]
The individual application bottles are stored within the main app location. How can you create a bottle in another location? e.g. a secondary storage hdd?

**Comment by mirkobrombin** [comment]
This will be possible in Bottles v4.

**Comment by mirkobrombin** [comment, marked as answer]
Ref: https://github.com/bottlesdevs/Bottles/issues/133
